fe1-web/res/strings.ts

```diff
@@ -29,7 +29,7 @@ const STRINGS = {
 
   /* --- Witness Navigation Strings --- */
   witness_navigation_tab_home: 'Witness',
-  witness_navigation_tab_video: 'Witness Video',
+  witness_navigation_tab_video: 'WitnessScreen Video',
```

Comment:
Was this by design? I believe this string does show up in the user interface.

Reply:
If relevant, I'll let you fix this in a subsequent PR, merging this one for now 👍